### ISSUE TITLE
Connection instable

### ISSUE THREAD

**Issue by kpagnat** [cpebit/aawireless-issues#161]
**Describe the bug**
Newly owner of the AAWireless, I made directly the update to 2.0.2 on first launch.
After that, I experienced many disconnects (sometimes every 2 minutes, sometimes after 30 minutes, it varies).
In the last days, I was having issues connecting to AndroidAuto on first start (many connect/disconnect).

After many tests, I found out that Spotify was the problem.
If Spotify is launched on my phone before starting the car, it will fail or disconnect just after connecting.
If Spotify is force closed, the connection is very quick (about 15seconds).
When everything works well and I launch Spotify from my phone (because Android Auto says Spotify doesn't work), then after a few minutes, it will disconnect...

**Device name (AAWireless/AndroidAuto-AAWxxxxxxx) (please complete the following information):**
 - AAWireless AAWG2sPnOwD

**Car (please complete the following information):**
 - Toyota CHR (2021)

**Headunit (please complete the following information):**
 -  Android Auto

**Smartphone (please complete the following information):**
 - Xiaomi Mi 11 Lite 5G

Is there something I can do ? Maybe it is about privileges from Spotify application ?

Thanks

**Comment by Snirpo** [comment]
Hmm, never heard this before. Can you check if the same thing happens with "passthrough" mode enabled?

**Comment by kpagnat** [comment]
Thanks for your quick answer.
In fact yes, I already tested this case. 
What I have done also is factory reset and using Dongle Mode too. But none of these work.

**Comment by kpagnat** [comment]
Okay. I'm back with some news.
It is definitively NOT Spotify's fault.
And the passthrough mode doesn't change anything. 
I didn't any have any trip without at least one disconnection (very annoying).

What can I do now ?
Thanks

**Comment by Snirpo** [comment]
I notice you have an Xiaomi phone, can you check if this is happening: https://www.youtube.com/shorts/gnlF6crZvJc ? If so AA dies because of phone loosing mobile connection. This is an issue with Xiaomi phones, there's nothing we can do about it unfortunately.

**Comment by Snirpo** [comment]
Closing because of no activity. Please open a new issue if this problem persists.

**Comment by kpagnat** [comment]
Hello,
You taked 16 days to answer my questions. And then after only 2 days you close the issue ?
I hoped to be better served for a product bought more than 100€.
Best regards.

**Comment by Snirpo** [comment]
I'm just closing old issues. This is no support desk, if you want immediate answers please contact support@aawireless.io
